app/scripts/modules/core/src/notification/types/googlechat/additionalFields.html

```diff
@@ -0,0 +1,12 @@
+<div class="form-group row">
+  <div class="col-sm-3 sm-label-right">Chat Webhook URL</div>
+  <div class="col-sm-9">
+    <input type="text"
+           name="address"
+           class="form-control input-sm "
+           ng-model="vm.notification.address"
+           placeholder="URL starts with https://chat.googleapis.com/v1/spaces/"
+           ng-required="!vm.notification.cc.length">
```

Comment:
i think you can replace this with just `required`.  The `cc` field is email-specific I believe.